cloud/pkg/cloudstream/iptables/iptables.go

```diff
@@ -182,7 +161,7 @@ func (im *Manager) getAddedAndDeletedCloudCoreIPPort() ([]string, []string, erro
 }
 
 func (im *Manager) getLatestTunnelPortRecords() (*TunnelPortRecord, error) {
-	configmap, err := im.cmLister.ConfigMaps(constants.SystemNamespace).Get(modules.TunnelPort)
+	configmap, err := kubeClient.CoreV1().ConfigMaps(constants.SystemNamespace).Get(context.Background(), modules.TunnelPort, metav1.GetOptions{})
```

Comment:
Thanks for the fixing, if I remember correvt it is possible to use the informer to list-watch one named k8s object like the configmap here @wuxs PTAL